db/migrate/20230117220305_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.text :bio
+      t.integer :posts_counter
+
+      t.timestamps
```

Comment:
- [ ] Kindly add the `photo` field to the `user` migration as it is part of the requirements